raiden-ts/src/types.ts

```diff
@@ -23,6 +24,8 @@ interface Info {
 export interface ContractsInfo {
   TokenNetworkRegistry: Info;
   ServiceRegistry: Info;
+  UserDeposit: Info;
+  OneToN: Info;
```

Comment:
I'd say there's no need to require `OneToN` info on this interface, which needs to be fullfilled by anyone instantiating a Raiden instance. Instead, it can be pulled in from `UserDepositContract.functions.one_to_n_address`, so one less source of inconsistency to need to check from there. Example:
https://github.com/raiden-network/light-client/blob/27c60e432ac9ab978ebf3b3a75aa72917f0a35f9/raiden-ts/src/path/utils.ts#L48-L51

Reply:
Thanks, I missed that completely. I will adjust accordingly